modules/mono/glue/GodotSharp/GodotSharp/Core/SignalAwaiter.cs

```diff
@@ -9,6 +9,7 @@ public class SignalAwaiter : IAwaiter<Variant[]>, IAwaitable<Variant[]>
         private bool _completed;
         private Variant[] _result;
         private Action _continuation;
+        private TaskCompletionSource<Variant[]> _taskCompletionSource = new();
```

Comment:
Where is this type coming from? Did you forget to commit a file?

Reply:
Oh, sorry, it's my first time creating a pull request, I forgot to add the include